### ISSUE TITLE
An error has occurred with Pandora/Audio Streaming failed

### ISSUE THREAD

**Issue by Carebears4lyf** [HermesApp/Hermes#330]
This is popping up consistently now, if it doesn't fail, songs will stutter every so often. 

I checked my internet speed and its as follows: Latency 29ms/download 59.01/upload 11.79

I have closed out of Hermes, and reopened. Adjusted audio quality from high to low. rebooted computer. Nothing has corrected this issue. This just began today. I will try removing and reinstalling after posting this as the final straw, in the meantime, any suggestions?

Thank you in advance.

**Comment by sirkitree** [comment]
This is happening to me consistently now as well, can't access any music at all through the app.

Even more interesting is that I can't access the main website on my mac either, but works perfectly on my windows machines. Even after connecting to a VPN. 🤔 I can't imagine OSX doing something to block this app somehow, but ... maybe?

I tried downloading the latest v1.3.1 from GitHub as well to no avail, still displays the "An error has occurred... " message. 

![img](https://d1ro8r1rbfn3jf.cloudfront.net/ms_3211/QbvoLweeFGfwxjwqsIj2qCrCMgUnnO/Hermes%2B2018-02-14%2B07-12-46.png?Expires=1518703991&Signature=Pz0XPnTGnJIRc7bRC~tnwGunCNrg-RoFOBYZHMSPxAF8Jrn1UnUvkHZuYTAdlktyZP5ql7YDzu3~QVpPNTz9ZZWgY8QnohN8djSoONfaHdjS6eMNcN2sCeC6hW00SZzHkXoaQ2yGE4waDP5vyyYuIaeBursgwo7YzZOW67C0oJt6Rgc0nEfXCvVhVsIgaDXiFwDWLIK2BvSN94M6lupoF78EZKYIbJ-11Z7aGyirJTIi6~YvuHgTTFNoVYK18M5M0c0pBlKChkkUhSB9RQTara~XBl6SWmm8VpAIoi-oci9FK~N3B5F~UIyvcjioSVRk6W4R31QKb5ABb6U6Oxl2MQ__&Key-Pair-Id=APKAJHEJJBIZWFB73RSA)

**Comment by Carebears4lyf** [comment]
I finally gave up and left it alone for a few days. When I was curious, upon reopening, it worked through the bug. Not sure what caused the bug, nor what cleared it up, but it is working like it used to.

**Comment by sirkitree** [comment]
Tried it again just now with the same negative result unfortunately.

**Comment by Carebears4lyf** [comment]
Looking at settings, mine are set at medium quality. Using system proxy settings. I don't think any other boxes checked or unchecked would cause the glitch?

**Comment by sirkitree** [comment]
I've replicated these settings with the same result.

**Comment by Carebears4lyf** [comment]
Running version 10.13.3 on Mac?

**Comment by ars292** [comment]
I've found that if it clears up for a bit it will come back eventually.

**Comment by Carebears4lyf** [comment]
When it stalls, was it at the 16-17 second marker, then at around 40 second, then crashing around 60 second? That's what mine consistently did. 

I'm nervous to start messing around with settings lest it causes it to happen again. I'm sorry friends, truly am

**Comment by ars292** [comment]
Yes that's exactly what it's doing. Except mine hasn't been crashing lately, just stuttering.

**Comment by sirkitree** [comment]
> Running version 10.13.3 on Mac?

I was on 10.12 - upgrading to 10.13 fix this and hermes works again :) Thank for you help!

**Comment by Carebears4lyf** [comment]
Good! Phew, glad we could figure this out together lol. 

Ars292, I would check to make sure you are up-to-date on your Mac then. Looks like an incompatibility issue with Hermes and OS.

**Comment by ars292** [comment]
I'm running 10.13.3

**Comment by Carebears4lyf** [comment]
Did you completely remove uninstall and all breadcrumbs before reinstalling too?

**Comment by ars292** [comment]
How can I make sure it's completely removed? I'm relatively new to Mac

**Comment by Carebears4lyf** [comment]
In that case, I would install an uninstaller to do that. There is a manual way, but if you delete the wrong item, it could mean a bad day for you

**Comment by ars292** [comment]
I actually had an app called AppCleaner which I used on Hermes. I rebooted, reinstalled, logged into pandora, and now I get the audio streaming failed error after about 10 seconds.

**Comment by wduquette** [comment]
Started seeing this two days ago, after having used Hermes without error for several months.  I'm running Hermes 1.3.1.  I'm on a fast stable network.  The old Pandora desktop app works fine, as does accessing Pandora through the browser.  The problem is intermittent.  Sometimes restarting helps.  Sometimes switching to a different station helps.  Sometimes it will run for hours, and sometimes I'm getting errors every few minutes.  Annoying.

**Comment by kltrout310** [comment]
Constantly receiving Internal Pandora Error.  is there a fix
